Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/GGVPointer.cs

```diff
@@ -81,6 +81,8 @@ public string PointerName
 
         public IMixedRealityCursor BaseCursor { get; set; }
 
+        public IMixedRealityCursor GazeCursor { get { return gazeProvider.GazePointer.BaseCursor; } }
```

Comment:
Why not just have the BaseCursor getter return gazeProvider.GazePointer.BaseCursor?

Reply:
I wasn't 100% certain on this. Is it always true that a GGVPointer will not have BaseCursor set?